.github/workflows/ci.generate.ts

```diff
@@ -601,9 +611,7 @@ const ci = {
           if: [
             "(matrix.job == 'test' || matrix.job == 'bench') &&",
             "matrix.profile == 'release' && (matrix.use_sysroot ||",
-            "(github.repository == 'denoland/deno' &&",
-            "(github.ref == 'refs/heads/main' ||",
-            "startsWith(github.ref, 'refs/tags/'))))",
+            "(github.repository == 'denoland/deno')))",
```

Comment:
The `skip_pr` flag already controls this, so this change allows us to test release builds without pushing to main